app/models/grda_warehouse/cas_project_client_calculator/mdha.rb

```diff
@@ -0,0 +1,21 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+require 'memoist'
+module GrdaWarehouse::CasProjectClientCalculator
+  class Mdha
+    def value_for_cas_project_client(client:, column:)
+      current_value = client.send(column)
+      return current_value unless column.to_sym == :match_group
+
+      # return 1 if has encampment decomissioning flag
+
+      return 2 if client.veteran?
+
+      return 3
```

Comment:
I think I'd rather see this as an explicit `if column.to_sym == :match_group` and the else can be  `current_value = client.send(column)`.  Possibly it's worth adding `private def match_group` so you can just call `client.send(:match_group)` I just don't like that `value_for_cas_project_client` has what appears to be default return values that are  for `match_group`